packages/frontend/src/components/common/GoogleLoginButton/index.tsx

```diff
@@ -44,3 +44,43 @@ export const GoogleLoginButton: React.FC<{ inviteCode?: string }> = ({
         </GoogleLoginWrapper>
     );
 };
+
+export const OktaLoginButton: React.FC<{ inviteCode?: string }> = ({
+    inviteCode,
+}) => {
+    const { health, showToastError } = useApp();
+    const flashMessages = useFlashMessages();
+
+    if (!health.data?.auth.okta.enabled) {
+        return null;
+    }
+
+    // TODO: will this happen twice if two buttons mounted?
+    // use in app provider?
```

Comment:
This would only happen if we support google and okta simultaneously. If we move this to the app provider it would make a GET every 250ms. Can the flash message have a issuer and we can filter it in this component ?

Reply:
Cool idea with filtering. Although I'm not actually sure when anybody will have both active